### ISSUE TITLE
Code 3

### ISSUE THREAD

**Issue by talsass** [talsass/CODE-3#1]
<!DOCTYPE html>
<html lang="fr">
<head>

     <meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<meta name="description" content="initiation au framework bootstrap">
	
	<link rel="stylesheet" type="text/css" href="inc/css/bootstrap.css">
	<link rel="stylesheet" type="text/css" href="inc/css/monstyle.css">
	<title> My bootstrap Page </title>
</head>

			<body>
			<header id="entetePrincipal">
	<div id="logo"><a href="#"><img src="inc/img/logo.gif" alt="Courchevel" /></a></div>

	
				<div class="container">
				<header>

<button type="button" class="btn btn-primary">primary</button>
<button type="button" class="btn btn-default" aria-label="Left Align">
  <span class="glyphicon glyphicon-home" aria-hidden="true"></span>
</button>

<button type="button" class="btn btn-primary btn-lg" data-toggle="modal" data-target="#myModal">
  Restaurant
</button>

<div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
  <div class="modal-dialog" role="document">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
        <h4 class="modal-title" id="myModalLabel">Sortir</h4>
      </div>
      <div class="modal-body">
        <h1>les pistes de Courchevel</h1>
        <article>
        	<p>
        		L’œil de Bœuf vous accueille tous les jours, sur sa terrasse ensoleillée au pied de la piste des Tovets, dans son petit chalet de montagne, pour déguster des viandes et des poissons de qualité, grillés devant vous au feu de bois, ainsi qu'un grand choix d'entrées et de desserts maison.
        	</p>
        </article>
      </div>
      <div class="modal-footer">
        <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
        <button type="button" class="btn btn-primary">Save changes</button>
      </div>
    </div>
  </div>
</div>

				
				<nav class="navbar navbar-default">
  <div class="container-fluid">
    
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="#">Brand</a>
    </div>

    
    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
      <ul class="nav navbar-nav">
        <li class="active"><a href="#">Link <span class="sr-only">(current)</span></a></li>
        <li><a href="#">Link</a></li>
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Dropdown <span class="caret"></span></a>
          <ul class="dropdown-menu">
            <li><a href="#">Action</a></li>
            <li><a href="#">Another action</a></li>
            <li><a href="#">Something else here</a></li>
            <li role="separator" class="divider"></li>
            <li><a href="#">Separated link</a></li>
            <li role="separator" class="divider"></li>
            <li><a href="#">One more separated link</a></li>
          </ul>
        </li>
      </ul>
      <form class="navbar-form navbar-left">
        <div class="form-group">
          
        </div>
        
      </form>
    
       
        
         
          <ul class="dropdown-menu">
            <li><a href="#">Action</a></li>
            <li><a href="#">Another action</a></li>
            <li><a href="#">Something else here</a></li>
            <li role="separator" class="divider"></li>
            <li><a href="#">Separated link</a></li>
          </ul>
        </li>
      </ul>
    </div>
  </div>
</nav>

					
				</header>
				</div>
				<div class="row">
				<div class="col-lg-12">
				<section>
				     <h1>Séjours à Courchevel</h1>

				     <article class="col-lg-6">
				     <img class="photo" src="inc/img/carte.jpg">
				     	
				     	<P>Bali est une île d'Indonésie située entre les îles de Java et de Lombok. Elle fait partie des petites îles de la Sonde. Sa superficie est de 5 637 km2. La population de l'ensemble des îles de la province de Bali était de 3 890 000 habitants en 2010, soit une densité de 690 habitants/km2. Administrativement, l'île fait partie de la province du même nom et accueille sa capitale (ibu kota), Denpasar, dans le sud de l'île</P>

				     </article>
					
					<article class="col-lg-6">
					<p>L'île de Bali se trouve à 3,2 km à l'est de Java et approximativement 8 degrés en dessous de l'Équateur. Bali et Java sont séparées par le détroit de Bali. D'est en ouest, l'île mesure approximativement 153 km de large. Elle s'étend sur environ 112 km du nord au sud. Sa superficie est de 5 637 km2.</p>

<p>La chaîne montagneuse du centre de Bali comprend plusieurs pics de plus de 2000 mètres d'altitude. Le plus haut est l'Agung (3142 m), un volcan actif baptisé la « mère montagne ». La chaîne s'élève du centre vers l'est, avec l'Agung dominant à l'extrême est. La nature volcanique de Bali contribue à son exceptionnelle fertilité et ses hautes chaînes montagneuses provoquent les fortes précipitations favorisant la forte production du secteur agricole. La vaste zone descendant du côté sud des montagnes est consacrée à la culture du riz. Les pentes du côté nord descendent plus fortement vers la mer. C'est le principal secteur de la production de café de l'île où l'on trouve également des légumes et du bétail. Le fleuve le plus long, la rivière Ayung (en), coule sur approximativement 75 km. Bali n'a pas de voies navigables importantes. La rivière Ho est cependant empruntées par de petits sampans.</p>
						
						</article>

						<article >
						<p class="col-lg-3"> L'île est entourée de récifs coralliens. Les plages du sud sont de sable blanc quand celles du nord et de l'est sont de sable noir. Les plages de sable noir entre Pasut et Klatingdukuh ont été développées pour le tourisme mais en dehors de celles proches du temple de Tanah Lot, elles ne sont pas encore utilisées de manière significative.</p>

<p class="col-lg-3">La plus grande ville de l'île est la capitale provinciale Denpasar située près de la côte sud. Sa population était d'environ 491 500 habitants en 2002. La seconde plus grande ville de Bali est l'ancienne capitale coloniale Singaraja, sur la côte nord, peuplée de près de 100 000 personnes. Les autres villes importantes sont la station balnéaire de Kuta, pratiquement dans la zone urbaine de Denpasar, et Ubud, au nord de Denpasar, connue comme le centre culturel de l'île.</p>
<p class="col-lg-3">
Trois petites îles proches situées au sud-est font administrativement partie du Kabupaten de Klungkung : Nusa Penida, Nusa Lembongan et Nusa Ceningan. Elles sont séparées de Bali par le détroit de Badung.</p>

<p class="col-lg-3">À l'est, le détroit de Lombok sépare Bali de Lombok et marque la division biogéographique entre la faune de l'écozone indomalaise et la faune distinctement différente d'Australasie. La transition est connue comme la ligne Wallace, du nom d'Alfred Russel Wallace qui, le premier, a proposé une zone de transition entre ces deux biomes majeurs. Lorsque le niveau de la mer baissa à la période glaciaire du Pléistocène, Bali fut reliée à Java et à Sumatra et à la partie continentale de l'Asie, accueillant ainsi la faune asiatique mais les eaux profondes du détroit de Lombok continuaient à maintenir Lombok et les îles de la Sonde isolées.</p>
							

						</article>
						<article >
						<img class="photo" src="inc/img/sky.jpg">
						<p class="col-lg-3"> L'île de Bali, comme la plupart des îles de l'archipel indonésien, est le résultat de la subduction tectonique de la plaque australienne sous la plaque eurasiatique. Le plancher océanique tertiaire, fait d'anciens dépôts marins tels que l'accumulation de récifs coralliens, fut soulevé au-dessus du niveau de la mer par la subduction. Des couches de calcaire du tertiaire soulevés du plancher océanique sont encore visibles dans des zones comme la péninsule de Bukit avec l'énorme falaise d'Uluwatu, ou au nord-est de l'île dans la péninsule de Prapat Agung1.</p>

<p class="col-lg-3">La déformation locale de la plaque eurasienne créée par la subduction a provoqué le fissurage de la croûte, conduisant à l'apparition de phénomènes volcaniques. Une chaîne de volcans parcourt la partie nord de l'île, dans un axe ouest-est où la partie ouest est la plus ancienne et la partie est la plus récente1. Le volcan actif le plus haut est un stratovolcan actif, l'Agung, à 3 142 mètres d'altitude.</p>

<p class="col-lg-3">L'activité volcanique a été intense au cours des âges et la plupart de la surface de l'île (en dehors des péninsules de Bukit et de Prapat Agung) a été recouverte par du magma volcanique. Certains de ces dépôts, de plus d'un million d'années, subsistent, tandis que la plupart de la partie centrale de l'île a été recouverte par des dépôts volcaniques récents, de moins d'un million d'années, dont certains champs de lave très récents dans le nord-est en raison de l'éruption catastrophique de l'Agung en 19631. L'activité volcanique, du fait de l'épais dépôt de cendres et de la terre fertile qu'il génère, sera un facteur important pour la prospérité agricole de l'île1.</p>

<p class="col-lg-3">En bordure de la subduction, Bali est aussi au bord du plateau continental de Sunda, juste à l'ouest de la ligne Wallace et fut un temps reliée à l'île voisine de Java, en particulier lors de l'abaissement du niveau de la mer pendant les périodes glaciaires. Sa faune et sa flore sont ainsi asiatiques2</p>
							

						</article>

				</section>
					

				</div>
					

				</div>
		
				

	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>			
<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>

			</body>
